src/commands/math/advancedSymbols.ts

```diff
@@ -0,0 +1,339 @@
+/************************************
+ * Symbols for Advanced Mathematics
+ ***********************************/
+
+LatexCmds.notin =
+LatexCmds.cong =
+LatexCmds.equiv =
+LatexCmds.oplus =
+LatexCmds.otimes = (latex:string) => new BinaryOperator('\\'+latex+' ', '&'+latex+';');
```

Comment:
I know this isn't particularly specific to this changeset, as the previous code behaved similarly, but the bindBinaryOperator method returns a function that takes no parameters and produces a BinaryOperator. It's a little weird that either of these signatures is fine to put into the LatexCmds map/object.

Also the general pattern of hiding data inside of these "factory methods" caused me some grief trying to write code that applied to all of the operators. In particular this pattern of defining totally different symbols with the same line of code and then relying on later code to pass back in the right latex made determining the information I needed seemingly impossible from where I was working. I ended up making a minor refactor here that I think makes the code clearer, and avoids this overloading of the LatexCmds data structure hold both paremeterless functions and those that require passing in the latex. https://github.com/mathquill/mathquill/pull/960/files#diff-a4c52f3aefd5adb5415b14a7e081bc82f379b5bf64b7fe889794beea20534b83R5

Reply:
Agree that that's a good refactor